Descriptor/OldEthnicity.metaed

```diff
@@ -0,0 +1,3 @@
+Descriptor OldEthnicity
```

Comment:
Old Ethnicity was removed with DS 5.0, but TPDM still uses it on candidate and this is a non-breaking release, so adding the descriptor to TPDM, you'll see its use in Candidate